Cargo.toml

```diff
@@ -27,10 +27,14 @@ byteorder = "1.2.6"
 log = "0.4"
 failure = "0.1"
 failure_derive = "0.1"
+clap = { version = "2.32", features = ["yaml"] }
```

Comment:
should this not be added to `[dev-dependencies]` or even maybe a better field. Since we are a library not an application.

Reply:
We are both